frontend/src/components/Overlay/MultipageOverlay.tsx

```diff
@@ -39,16 +39,14 @@ function MultipageOverlay({
 
 	let speakerImage;
 
-	if (imgSource[currentPage] === 'Handler') {
+	if (pages[currentPage].imageName === 'Handler') {
 		speakerImage = Handler;
-	} else if (imgSource[currentPage] === 'ScottBrew Manager') {
+	} else if (pages[currentPage].imageName === 'ScottBrew Manager') {
 		speakerImage = Manager;
-	} else if (imgSource[currentPage] === 'ScottBrew Lawyer') {
+	} else if (pages[currentPage].imageName === 'ScottBrew Lawyer') {
 		speakerImage = Lawyer;
-	} else if (imgSource[currentPage] === 'SpyLogic') {
-		speakerImage = BotAvatarDefault;
 	} else {
-		speakerImage = '';
+		speakerImage = BotAvatarDefault;
 	}
```

Comment:
The only thing I'd consider changing here is forcing the components that are using MultipageOverlay to import their own images and pass them in, instead of passing in a name and using this if...else block to pick the correct image.

The reasoning being, now MultipageOverlay needs knowledge of all the images that could possibly be requested by users of this component, so if you want a new image, you'd need to make code changes here and in the parent component.

Reply:
ah yes that makes sense, didn't even know that would work but it does :)

Reply:
It works because when you import an image file in Vite, you actually get a URL to that image, which is a string. The image itself is a static asset:
https://vitejs.dev/guide/assets#static-asset-handling

For this reason, it might be clearer to call the prop "imageUrl" rather than "imageName".